submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,87 @@
+const url = 'https://randomuser.me/api/?results=12';
+const content = document.querySelector('.articleBlock');
+const form = document.forms;
+let friends = [];
+let friendsCopy = [];
+
+const data = (url) => fetch(url)
+    .then(handleErrors)
+    .then(res => res.json())
+    .then(res => res.results)
+    .then(res => {
+        friends = [...res];
+        friendsCopy = [...friends];
+        renderAllItemsToPage(friends);
+    });
+
+function handleErrors(res) {
+    if (!res.ok) {
+      throw Error(res.statusText);
+    }
+    return res;
+}
+
+document.querySelector('.asideBlock').addEventListener('click', sortBy);
+function sortBy({target}) {
+    const id = target.id;
+
+    switch(id) {
+        case 'ageMore':
+            friendsCopy = friendsCopy.sort((a, b) => a.dob.age - b.dob.age);
```

Comment:
Look at the functions that you use for sorting. They are identical, except you change parameter order. Maybe abstract it to not repeat yourself.